src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -159,13 +164,18 @@ private Content generateHtml(SlingHttpServletRequest request, Resource resource)
         return c;
     }
 
-    private void cacheContent(SlingHttpServletRequest request, Module module, Content content) {
+    private void cacheContent(final SlingHttpServletRequest request, final Module readOnlyModule, final Content content) {
```

Comment:
Just a good practice. Parameters should not be changed in the middle of the method so marking them final is a way to ensure that. It also helps if I try to use them in a lambda or anonymous class.

Reply:
Hmm... ok, I agree with your logic, but it's uncommon (both generally and in this project), so that made me think it had a purpose. But it doesn't hurt anything to have it.